Revise wording about shared user accounts when using SDK (#52986)
metabase/metabase: docs/embedding/sdk/dashboards.md

| onLoadWithoutCards           | `(dashboard: Dashboard \| null) => void`        | Event handler that triggers after dashboard loads, but without its cards - at this stage dashboard title, tabs and cards grid is rendered, but cards content is not yet loaded.                                                                                                                                                  |
| renderDrillThroughQuestion\* | `() => ReactNode`                               | A react component that renders [a question's layout](#customizing-drill-through-question-layout) shown after drilling through a question or clicking on a question card in the dashboard.                                                                                                                                        |

_\* Not available for `StaticDashboard`._

> -\*\* Combining `initialParameters` and `hiddenParameters` with the purpose of decluttering the interface is fine, but if done filter data on the front-end, it's considered risky and against our terms. [More info](../authentication#security-warning-each-end-user-must-have-their-own-metabase-account).

Reviewer: ```suggestion
-\*\* Combining `initialParameters` and `hiddenParameters` to filter data on the frontend is a [security risk and is against our terms](../authentication#security-warning-each-end-user-must-have-their-own-metabase-account). Combining `initaliParameters` and `hiddenParaters` to declutter the user interface is fine. 
```
Frontloads the warning.

> -\*\* Combining `initialParameters` and `hiddenParameters` to filter data on the frontend is a [security risk and is against our terms](../authentication#security-warning-each-end-user-must-have-their-own-metabase-account). Combining `initaliParameters` and `hiddenParaters` to declutter the user interface is fine. 

Author: ```suggestion
-\*\* Combining `initialParameters` and `hiddenParameters` to filter data on the frontend is a [security risk and is against our terms](../authentication#security-warning-each-end-user-must-have-their-own-metabase-account). Combining `initialParameters` and `hiddenParaters` to declutter the user interface is fine. 
```
